pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -5233,42 +5241,41 @@ protected void internalGetReplicatedSubscriptionStatus(AsyncResponse asyncRespon
         });
     }
 
+    private CompletableFuture<Map<String, Boolean>> getReplicatedSubscriptionStatusFromBroker(TopicName partitionName,
```

Comment:
`partitionName` here is confusing. It also applies to non-partitioned topic.

Reply:
Changed  to `localTopicName`.